web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -118,6 +118,10 @@ export const createAppConfigSchema = (platform: Platform) => {
       .describe(
         'Skips the confirmation prompt for headless login approval and instead prompts for WebAuthn immediately.'
       ),
+    'ssh.noResume': z
```

Comment:
We need to update the docs too: https://github.com/gravitational/teleport/blob/499f2cfada041f1a1e5c65ddf4bb95a45d492bfa/docs/pages/connect-your-client/teleport-connect.mdx#L375-L393

Reply:
I'll make a note to update the Connect docs when writing the docs around connection resumption in general.